README.md

```diff
@@ -60,54 +57,48 @@
+./build ${platform}-${feature}_${modifier}
 ```
 
-The build script will automatically fetch the necessary builder container and execute all the required build steps internally. By default, the build system employs rootless podman. However, you can configure it to use a different container engine by utilizing the `--container-engine` flag.
-
-## Build Requirements
+Where:
+- `${platform}` denotes the desired platform (e.g., kvm, metal, aws).
+- `${feature}` represents a specific feature from the `features/` folder.
+- `${modifier}` is an optional modifier from the `features/` folder, prefixed with an underscore "_".
 
-To successfully build the project, ensure the following requirements are met:
-
-- **Memory:** The build process may require up to 8GiB of memory, depending on the selected targets. If your system has insufficient RAM, configure swap space accordingly.
-- **Container Engine:** The Builder has minimal dependencies and only requires a working container engine. It is recommended to use rootless Podman. Please refer to the [Podman rootless setup guide](https://github.com/containers/podman/blob/main/docs/tutorials/rootless_tutorial.md) for instructions on setting it up.
-
-### SELinux
-
-If you intend to build targets with the `_selinux` feature some additional requirements apply to the build machine.
-Building the `_selinux` feature will not work on machines running in SELinux enforcing mode. Ideally you should build on a build machine with SELinux disabled, but if you want to build with SELinux in permissive mode this can be achieved by running build as root with the `--privileged` flag.
+You can combine multiple platforms, features, and modifiers as needed.
 
-i.e.: `sudo ./build --privileged ${target}`
+Example:
+```shell
+./build kvm-python_dev
+```
 
-## Secureboot
+The build script fetches the required builder container and manages all internal build steps. By default, it uses rootless podman, but you can switch to another container engine with the `--container-engine` flag.
 
-If you intend to build targets with the `_secureboot` feature, you must first build the secureboot certificates.
-Run the command `./cert/build` to generate the secureboot certificates.
+### Build Requirements
 
-By default, the command uses local files as the private key storage. However, you can configure it to use the AWS KMS key store by using the `--kms` flag. Note that valid AWS credentials need to be configured using the standard AWS environment variables.
+- **Memory:** The build process may require up to 8GiB of memory, depending on the selected targets. If your system has insufficient RAM, configure swap space accordingly.
+- **Container Engine:** The Builder has minimal dependencies and only requires a working container engine. It is recommended to use rootless Podman. Please refer to the [Podman rootless setup guide](https://github.com/containers/podman/blob/main/docs/tutorials/rootless_tutorial.md) for instructions on setting it up.
```

Comment:
minor comment: when I last tried to setup rootless podman on an ubuntu server, I found their docs not that good and gave up after some attempts. I fully support preferring rootless podman, and I think it's great that we support it, but maybe it would be good to mention here that a default podman install with sudo will also just work.